src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php

```diff
@@ -606,6 +606,103 @@ protected function typeMacAddress(Fluent $column)
     }
 
     /**
+     * Create the column definition for a geometry type.
+     *
+     * @param  \Illuminate\Support\Fluent  $column
+     * @throws \Exception
+     */
+    protected function typeGeometry(Fluent $column)
+    {
+        throw new \Exception('Geometry data type not supported for current database engine.');
```

Comment:
Didn't know what type of exception would be better to throw here.

Reply:
I guess `RuntimeException`. 👍 

Eventually Laravel could implement a generic `NotImplementedException` ([like this one](https://github.com/crazycodr/standard-exceptions/blob/master/Exceptions/Operation/NotImplementedException.php)) for all those cases where something is not implemented by design to better signal these limitations. 😃

Reply:
I've thought about `RuntimeException` too.